Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -285,7 +285,7 @@
 
 - type: entity
   name: pulse carbine
-  parent: [BaseWeaponBattery, BaseWeaponBatteryTransfer, BaseGunWieldable, BaseCentcommContraband]
+  parent: [BaseGunMeleeDisarm, BaseWeaponBattery, BaseWeaponBatteryTransfer, BaseGunWieldable, BaseCentcommContraband]
```

consider making long lists like this an actual list
```suggestion
  parent:
  - BaseGunMeleeDisarm
  - BaseWeaponBattery
  - BaseWeaponBatteryTransfer
  - BaseGunWieldable
  - BaseCentcommContraband
```